contrib/kind-helm.sh

```diff
@@ -73,6 +88,7 @@ usage() {
     echo "       [ -ikv | --install-kubevirt ]"
     echo "       [ -mne | --multi-network-enable ]"
     echo "       [ -wk  | --num-workers <num> ]"
+    echo "       [ -npz | --node-per-zone ]"
```

Comment:
to keep this as compatible with existing kind.sh, let's ALSO add `[-ic | --enable-interconnect]` flag, using the environment variable `OVN_ENABLE_INTERCONNECT
`
It will make it easier for folks to use kind-helm.sh from kind.sh

ref: https://github.com/ovn-org/ovn-kubernetes/blob/ff286af7048c286f77963490f4756516da6ee9b0/contrib/kind.sh#L79

https://github.com/ovn-org/ovn-kubernetes/blob/ff286af7048c286f77963490f4756516da6ee9b0/contrib/kind.sh#L397-L403

Reply:
... and another BIG reason for paying attention to OVN_ENABLE_INTERCONNECT comes from the fact that I think we really should add a new line in the github workflow to ensure ovn-ic is deployed with helm going forward. I will connect with you on that offline.

Reply:
ack, ported OVN_ENABLE_INTERCONNECT from kind.sh.